### PR TITLE
Use auto output for Adafruit BLE

### LINKED ISSUE #3577: Adafruit Bluefruit 32u4: OUT_AUTO not applied by default
Pretty much what #2229 describes - the keyboard is not applying out_auto by default and sends keycodes on both usb and bluetooth.

### PR DESCRIPTION
**Pull request by ecopoesis**
When using `MODULE_ADAFRUIT_BLE` both USB and Bluetooth output were used. This adds an `#IFDEF` to set output to `OUTPUT_AUTO` when this module is defined.

Fixes #3577

### CODE REVIEW

**Comment by drashna** [comment]
There are some merge conflicts here that need to be resolved.

File: quantum/quantum.c [hunk 1]
```diff
@@ -957,6 +957,9 @@ void matrix_init_quantum() {
   #ifdef RGB_MATRIX_ENABLE
     rgb_matrix_init();
   #endif
+  #ifdef MODULE_ADAFRUIT_BLE
```
Comment (drashna) [inline quantum/quantum.c]: Should this be defaulted to auto? 

I think it would be better to hide this behind a define, to configure it's status

**Comment by drashna** [comment]
Also, there are merge conflicts here.

**Comment by yanfali** [comment]
We've been getting reports that BLE has stopped working. Is this actually working still?

**Comment by skullydazed** [comment]
@ecopoesis We'd like to merge or close this PR. Are you going to continue working on this?

**Comment by ecopoesis** [comment]
I've change the define so it's now called `OUTPUT_AUTO_ENABLE`. What else is needed to get this merged?

This (and BLE in general) works in my branch: https://github.com/ecopoesis/qmk_firmware/tree/nek_type_a_auto

**Pull request merged**